### PR TITLE
Follow-up edits to PR#1816

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1816

### CODE REVIEW

File: architecture/additional_concepts/authorization.adoc [hunk 1]
```diff
@@ -475,53 +479,56 @@ Validates against all ranges.
...
+* *iscsi*
+* *glusterfs*
+* *persistentVolumeClaim*
+* *rbd*
+* *cinder*
+* *cephFS*
+* *downwardAPI*
+* *fc*
+* *configMap*
+*  ***
```
Comment (tpoitras) [inline architecture/additional_concepts/authorization.adoc]: does this render as 3 asterisks via our build process? Or do these need to be wrapped in backticks or escaped somehow?

File: architecture/additional_concepts/authorization.adoc [hunk 2]
```diff
@@ -475,53 +479,56 @@ Validates against all ranges.
...
+`***` is a special value to allow the use of all volume types.
 ====
 
 [NOTE]
 ====
-For backwards compatibility, the usage of `*allowHostDirVolumePlugin*` will override settings
-in the `*volumes*` field.  For example, if `*allowHostDirVolumePlugin*` is set to false but allowed
-in the `*volumes*` field the `*hostPath*` value will be removed from `*volumes*`.
+For backwards compatibility, the usage of `*allowHostDirVolumePlugin*` will
+override settings in the `*volumes*` field.  For example, if
```
Comment (tpoitras) [inline architecture/additional_concepts/authorization.adoc]: s/will override/overrides

File: architecture/additional_concepts/authorization.adoc [hunk 3]
```diff
@@ -475,53 +479,56 @@ Validates against all ranges.
...
+`*allowHostDirVolumePlugin*` is set to false but allowed in the `*volumes*`
+field the `*hostPath*` value will be removed from `*volumes*`.
```
Comment (tpoitras) [inline architecture/additional_concepts/authorization.adoc]: s/field the/field, then the

File: architecture/additional_concepts/authorization.adoc (hunk 1 above)
Comment (ahardin-rh) [inline architecture/additional_concepts/authorization.adoc]: It appears as a bolded asterisk in my local build

**Comment by ahardin-rh** [comment]
No rev history needed

**Pull request merged**